src/rollback_to_stable/rts_history.c

```diff
@@ -54,8 +55,16 @@ __wt_rts_history_delete_hs(WT_SESSION_IMPL *session, WT_ITEM *key, wt_timestamp_
         if (hs_tw->stop_ts <= ts)
             break;
 
-        if (!dryrun)
+        if (!dryrun) {
+            __wt_verbose_level_multi(session, WT_VERB_RECOVERY_RTS(session), WT_VERBOSE_DEBUG_3,
+              WT_RTS_VERB_TAG_HS_UPDATE_REMOVE
+              "deleting history store update with stop_timestamp=%s greater than "
```

Comment:
Print the entire time window here that is available with `hs_tw`.

Reply:
Done! [2ff1584](https://github.com/wiredtiger/wiredtiger/pull/8960/commits/2ff15849792bdb0a1de4f07f3f8cb187db8e72a2)